tests/serve_request.rs

```diff
@@ -25,6 +25,10 @@ fn serves_requests_with_no_options(tmpdir: TempDir) -> Result<(), Error> {
     let parsed = Document::from_read(body)?;
     for &file in FILES {
         assert!(parsed.find(|x: &Node| x.text() == file).next().is_some());
+
+        let resp = reqwest::blocking::get(format!("http://localhost:8080/{}", file))?
```

Comment:
For specific checks like this, I like seeing a comment as to _why_ we're checking for this particular thing.